bucket/qsv.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": "0.119.0",
+    "description": "A command line CSV data-wrangling toolkit",
+    "homepage": "https://github.com/jqnatividad/qsv",
+    "license": "Unlicense|MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/jqnatividad/qsv/releases/download/0.119.0/qsv-0.119.0-x86_64-pc-windows-msvc.zip",
+            "hash": "a33e3278d1bee72095250bda8cb07d0aed80b656dae4767cd1878c6aa4362ced"
+        },
+        "32bit": {
+            "url": "https://github.com/jqnatividad/qsv/releases/download/0.119.0/qsv-0.119.0-i686-pc-windows-msvc.zip",
+            "hash": "ae3149abec07ab73e826ee8ff7a497f2485dd2063d9a872f257ee66dd3bdc9d6"
+        }
+    },
+    "bin": "qsv.exe",
```

Comment:
I think we should add `qsvlite `and `qsvdp` to bin.

Also, do you know what purpose the nightly versions serve? They seem to have almost the same version strings.
```
❯ .\qsv_nightly.exe --version
qsv 0.119.0-mimalloc-apply;fetch;geocode;Luau 0.604;polars-0.35.4;self_update-8-8;6.29 GiB-2.67 GiB-607.35 MiB-7.87 GiB (x86_64-pc-windows-msvc compiled with Rust 1.74.0) prebuilt-nightly

❯ .\qsv.exe --version
qsv 0.119.0-mimalloc-apply;fetch;geocode;Luau 0.604;polars-0.35.4;self_update-8-8;6.29 GiB-2.69 GiB-546.75 MiB-7.87 GiB (x86_64-pc-windows-msvc compiled with Rust 1.74.0) prebuilt
```

Reply:
> I think we should add `qsvlite` and `qsvdp` to bin.

1. My fault, added both. Thank you for correction.

> Also, do you know what purpose the nightly versions serve? They seem to have almost the same version strings.

2. [Here](https://github.com/jqnatividad/qsv/blob/master/docs/PERFORMANCE.md#nightly-release-builds) is described how the "nightly" versions are different from original. 
I guess only the "stable" are in scope of Main bucket, the "nightly" could be added to the Versions bucket (in case of need).